modules/sdk/gradle-plugins-target-platform/CHANGELOG.markdown

```diff
@@ -13,14 +13,16 @@
 	- configurations added by [Liferay Gradle Plugins Test Integration]
 	- configurations added by the `java` plugin
 
-## 1.1.0 - *(Unreleased)*
+## 1.1.0 - 2018-07-03
 
 ### Added
 - [LPS-81530]: Add the ability to configure imported BOMs to manage Java
 dependencies and the various artifacts used in resolving OSGi dependencies.
+- [LPS-82491]: Add support for IntelliJ Idea plugin
```

Comment:
I think we have the convention to write something like this instead:
> the `idea` plugin.

Don't forget the period at the end.